onnxruntime/test/providers/cpu/tensor/tensor_op_test.cc

```diff
@@ -91,6 +91,39 @@ TEST(TensorOpTest, Reshape_WithOutAllowZero) {
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
 }
 
+TEST(TensorOpTest, Reshape_WithOutAllowZeroToDiffRank) {
+  OpTester test("Reshape", 14);
+
+  test.AddInput<float>("data", {2, 3, 12}, std::vector<float>(72, 1.0f));
+  test.AddInput<int64_t>("shape", {4}, {2, 3, 3, 4}, true);
+  test.AddAttribute<int64_t>("allowzero", 0);
+  test.AddOutput<float>("reshaped", {2, 3, 3, 4}, std::vector<float>(72, 1.0f));
+  // TensorRT doesn't support dynamic shape tensor for now
```

Comment:
what does "dynamic shape tensor" mean here? the shape input is an initializer.

Reply:
Oops -- the test was copied / templated from previous tests